WalletWasabi/CoinJoin/Client/CoinJoinProcessor.cs

```diff
@@ -47,7 +47,7 @@ private async void Synchronizer_ResponseArrivedAsync(object? sender, Synchronize
 
 					var txsNotKnownByAWallet = WalletManager.FilterUnknownCoinjoins(unconfirmedCoinJoinHashes);
 
-					using var client = Synchronizer.WasabiClientFactory.NewBackendClient();
+					using var client = Synchronizer.WasabiClientFactory.NewBackendClient(isolateStream: true);
```

Comment:
The request is for getting the unconfirmedCoinJoins from the backend and rebroadcast if needed. It is fine if that happens on the default stream. So this PR is not needed.
@nopara73 ?

Reply:
I'm not sure. I have rather reported that to make sure we did not introduce bad behavior.